data-lineage/src/main/java/com/wipro/ats/bdre/lineage/LineageMain.java

```diff
@@ -568,16 +445,13 @@ private void handleTokJoin(ASTNode nextNodeOfFrom) {
 				handleTokJoin(childAST);
 
 			} else {
-				System.out.println(childAST.toString() + " ::: " + childAST.getType());
+				LOGGER.info(childAST.toString() + " ::: " + childAST.getType());
 			}
 		}
 	}
 
 	private void handleTokTabRef(ASTNode nextNodeOfFrom) {
 		ASTNode inputTableNodeInTR = (ASTNode) nextNodeOfFrom.getChild(0);
-//		String table_name = (inputTableNodeInTR.getChildCount() == 1) ?
-//				getUnescapedName((ASTNode) inputTableNodeInTR.getChild(0)) :
-//				getUnescapedName((ASTNode) inputTableNodeInTR.getChild(0)) + "." + inputTableNodeInTR.getChild(1);
 		String table_name = (inputTableNodeInTR.getChildCount() == 1) ?
 				getUnescapedName((ASTNode) inputTableNodeInTR.getChild(0)) : getUnescapedName((ASTNode) inputTableNodeInTR.getChild(1)) ;
 		String table_alias = null;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this local variable name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS00117)